jujugui/static/gui/src/app/utils/zip-utils.js

```diff
@@ -28,8 +28,12 @@ YUI.add('zip-utils', function(Y) {
 
   // Configure the zip.js library. The workerScriptsPath must point to the
   // location of inflate.zip and deflate.zip.
-  zip.workerScriptsPath = 'static/gui/build/app/assets/javascripts/';
-
+  if (window.juju_config && window.juju_config.staticURL) {
+    zip.workerScriptsPath = window.juju_config.staticURL +
+        '/combo?app/assets/javascripts/';
```

Comment:
It seems so. I tried constructing the non combo URL and it didn't serve it up. 

Sent from a mobile device with a very small keyboard. Please forgive typos.




On Wed, Jul 13, 2016 at 5:11 PM -0400, "Jeff Pihach" <notifications@github.com> wrote:












In jujugui/static/gui/src/app/utils/zip-utils.js:
> @@ -28,8 +28,12 @@ YUI.add('zip-utils', function(Y) {
>  
>    // Configure the zip.js library. The workerScriptsPath must point to the
>    // location of inflate.zip and deflate.zip.
> -  zip.workerScriptsPath = 'static/gui/build/app/assets/javascripts/';
> -
> +  if (window.juju_config && window.juju_config.staticURL) {
> +    zip.workerScriptsPath = window.juju_config.staticURL +
> +        '/combo?app/assets/javascripts/';



Do we need to use the '/combo?' endpoint? 



—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.